### PR TITLE
added some tests to increase the sonar code cov

### PR DESCRIPTION
**Pull request by kurt-uk**
to increase the code cov, getter and setter tests were added

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Evoco Digital Services Ltd
```
Comment (stevewalton28) [inline LICENSE]: 2021

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Test Data Generator
+![TestData CI](https://github.com/digital-delivery-academy/simple-test-data-generator/workflows/TestData%20CI/badge.svg)
+
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=digital-delivery-academy_simple-test-data-generator&metric=coverage)](https://sonarcloud.io/dashboard?id=digital-delivery-academy_simple-test-data-generator)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=digital-delivery-academy_simple-test-data-generator&metric=alert_status)](https://sonarcloud.io/dashboard?id=digital-delivery-academy_simple-test-data-generator)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=digital-delivery-academy_simple-test-data-generator&metric=security_rating)](https://sonarcloud.io/dashboard?id=digital-delivery-academy_simple-test-data-generator)
+
+## Documentation
+- A full reference guide is here: https://digital-delivery-academy.github.io/selenium-pom-framework/index.html
```
Comment (stevewalton28) [inline README.md]: Well we don't have these do we?  Remove all refs to the Selenium framework in the README I think

File: src/test/java/uk/co/evoco/testdata/BankHolidaysTests.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class BankHolidaysTests {
+
+    private BankHolidays bankHolidays = new BankHolidays();
+
+    @Test
+    public void testGetAndSetLocale() {
+        bankHolidays.setLocale("scotland");
```
Comment (stevewalton28) [inline src/test/java/uk/co/evoco/testdata/BankHolidaysTests.java:15]: Could test for upper case, mixed case and no match to the ENUM here (e.g. UNITED_STATES)

**Review by stevewalton28 — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (kurt-uk) [reply LICENSE]: 👍🏼

File: README.md (hunk 2 above)
Comment (kurt-uk) [reply README.md]: removed 👍🏼 
Id need a description of what we wanna put in the readme though, could be best for you to write, or send some bullet points for me and ill juice them up a bit

File: .all-contributorsrc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      "name": "Kurt P",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/26488566?v=4",
+      "profile": "https://github.com/kurt-uk",
+      "contributions": [
+        "code"
+        ]
+       }
+  ],
+  "contributorsPerLine": 7,
+  "projectName": "selenium-pom-framework",
```
Comment (kurt-uk) [inline .all-contributorsrc]: change

**Review by stevewalton28 — APPROVED** [review]

**Pull request merged**